package.json

```diff
@@ -142,7 +143,8 @@
     "build:rollup": "rollup -c rollup.config.ts",
     "build:minify-browser": "terser -c -m --comments --source-map \"content='./dist/msRest.browser.js.map'\" -o ./dist/msRest.browser.min.js ./dist/msRest.browser.js",
     "build:test-browser": "webpack --config webpack.testconfig.ts",
-    "format": "prettier --write \"./**/*.ts\"",
+    "extract-api": "api-extractor run --local",
```

Comment:
should we also call this as part of `build`?

Reply:
added to `build:lib`